README.md

```diff
@@ -4,10 +4,33 @@ Welcome to the Paseo Action Submission (PAS) Repository, the central hub for all
 ## Purpose
 The Paseo Action Submission Repository serves as a structured platform for submitting, reviewing, and discussing proposals that impact the Paseo network's technical specifications, governance, features, and improvements. Our aim is to ensure that the Paseo testnet evolves in a way that meets the community's needs and adheres to high standards of quality and innovation.
 
+## PAS workflow
+The individuals involved in creating a PAS are:
+1. The author of the PAS
+2. the Paseo Governance Team (PGT) which are the signers on the sudo multisig
+
+When creating a PAS, please use the following statuses in the title of your PAS - this allows the community to see which proposals have been implemented and which are still in discussion. Please update the status with a PR. 
```

Comment:
Let's use Github Labels for this. It would be good that the text says that.

@carlosala could we restrict that only members of the repos can change the labels ?

Reply:
Labels are good - I guess the point was when you change the status it has to be a PR change, so that it needs to be approved and there is more of an audit trail.

Reply:
But Github PR status are not enough to track these status?
we would have: 
- Open: Proposed and under discussion PAS
- Merged: Approved and merged PAS
- Closed: Dissaproved PAS